apis.go

```diff
@@ -94,23 +94,25 @@ type OVNDBApi interface {
 	Execute(cmds ...*OvnCommand) error
 
 	// Get all logical switches
-	GetLogicSwitches() []*LogicalSwitch
+	GetLogicalSwitches() ([]*LogicalSwitch, error)
```

Comment:
I agree that GetLogicalSwitches is better. Just to make sure @lebauce won't be surprised of this API change.

Reply:
i don't think that this is problem, because this is already new version, so we don't break old.